web/src/beta/components/Modal/index.stories.tsx

```diff
@@ -25,26 +26,66 @@ const ModalContent: React.FC = () => {
   );
 };
 
+const TabContent: ReactNode = (
+  <>
+    <Text size="h1">Tab Title</Text>
+    <Text size="h5">
+      Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of
+      classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a
+      Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin
+      words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in
+      classical literature, discovered the undoubtable source.
+    </Text>
+  </>
+);
+
+const sidebarTab1: SidebarTab = {
+  id: "1",
+  label: "Tab1",
+  content: TabContent,
+  tabButton1: <Button text="Confirm" />,
+  tabButton2: <Button text="Cancel" />,
+};
+
+const sidebarTab2: SidebarTab = {
+  id: "1",
+  label: "Tab1",
```

Comment:
Update to be consistent. Also to avoid both tabs being selected at the same time

Reply:
sorry it seems last commit didn't sync right , it is already modified